editor/create_dialog.cpp

```diff
@@ -305,16 +323,19 @@ void CreateDialog::_configure_search_option_item(TreeItem *r_item, const StringN
 		r_item->set_text(0, p_type);
 		String script_path = ScriptServer::get_global_class_path(p_type);
 		Ref<Script> scr = ResourceLoader::load(script_path, "Script");
-		String suffix = script_path.get_file();
-		if (scr.is_valid() && custom_type_suffixes.has(p_type)) {
-			suffix = custom_type_suffixes.get(p_type);
-		}
-		if (!suffix.is_empty()) {
-			r_item->set_suffix(0, "(" + suffix + ")");
-		}
-
 		ERR_FAIL_COND(!scr.is_valid());
 		is_abstract = scr->is_abstract();
+
+		String tooltip = "Script path: %s";
```

Comment:
```suggestion
		String tooltip = TTR("Script path: %s");
```
The main purpose of TTR is to mark string for translation, hence it should be used only on string literals.